docs/process.rst

```diff
@@ -1982,7 +1982,7 @@ saveAs          A closure which, given the name of the file being published, ret
 enabled         Enable or disable the publish rule depending on the boolean value specified (default: ``true``).
 failOnError     When ``true`` abort the execution if some file can't be published to the specified target directory or bucket for any cause (default: ``false``)
 contentType     Allow specifying the media content type of the published file a.k.a. `MIME type <https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_Types>`_. If the boolean value ``true`` is specified the content type is inferred from the file extension (EXPERIMENTAL. Currently only supported by files stored on AWS S3. Default: ``false``, requires `22.10.0`` or later).
-tags            Allow the association of arbitrary tags with the published file e.g. ``tag: [FOO: 'Hello world']`` (EXPERIMENTAL. Currently only supported by files stored on AWS S3. Requires version ``21.12.0-edge`` or later).
+tags            Allow the association of arbitrary tags with the published file e.g. ``tags: [FOO: 'Hello world']`` (EXPERIMENTAL. Currently only supported by files stored on AWS S3. Requires version ``21.12.0-edge`` or later).
```

Comment:
From the source code it does seem like it should be `tags` 

https://github.com/nextflow-io/nextflow/blob/916dd1a4cc0a6425f3c7d86b5f228b71ed056d6e/modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy#L179-L177